### DISCUSSION TITLE
Hover Card inside Carousel

### DISCUSSION THREAD

**Discussion by inirizky** [shadcn-ui/ui#2847] [Q&A]
Is it possible to use Hover cards inside Carousel? because I struggled when I tried it

I am using ReactJS

**Comment by ZionLG** [comment]
I have the same Issue, It doesnt seem to work.

**Comment by ZionLG** [comment]
![image](https://github.com/shadcn-ui/ui/assets/36079911/5b1057b0-6c9c-4976-ab38-6afb928fd238)
the issue seems to be this class here ("overflow-hidden"). its used in the Carousel to hide the next/past items, but in turn also hides the hover.

**Reply by pedro-stark** [reply]
any known workaround?

**Comment by dvenomb98** [comment]
Same problem, didnt found solution yet, probably will create my own carousel.

**Comment by NhanNguyen11002** [comment]
still no update about this bug, does anyone make it through this bug yet  ??

**Comment by FredMT** [comment]
Ah, figured out an easier solution. 

Just include this line in hover-card.tsx (in your @components/ui) - ```const HoverCardPortal = HoverCardPrimitive.Portal```

Then wrap your HoverCardContent with it, like this :-

```
<HoverCardPortal container={document.body}>
    <HoverCardContent className="h-[300px] w-[400px]">
        <div>Hello World</div>
    </HoverCardContent>
</HoverCardPortal>
```

This way, it appears above all other elements, regardless of stacking order.

https://www.radix-ui.com/primitives/docs/components/hover-card#portal

**Reply by trancethehuman** [reply]
^ This fixed it for me. Thank you @FredMT

**Reply by osh6006** [reply]
Amazing! Thank you!!!! @FredMT
